easy-pbkdf2.js

```diff
@@ -203,7 +203,7 @@ EasyPbkdf2.prototype = {
 		var keyLength,
 			easyPbkdf2;
 
-		if ( !priorHash || !_.isString( priorHash ) ) {
+		if ( !priorHash || !( typeof priorHash === "string" ) ) {
```

Comment:
Unfortunately, this check would now fail if `priorHash` was created via `new String()`... and I don't want to introduce a potentially breaking change here. I am tempted to say that anyone creating strings this way deserves to have their code broken, I just can't do it. I wish I had written it this way from the start.

I'm curious, why do you want to remove lodash as a dep?